src/jsonapi.js

```diff
@@ -1,5 +1,5 @@
 import { createAction, handleActions } from 'redux-actions';
-import fetch from 'isomorphic-fetch';
+import 'isomorphic-fetch';
```

Comment:
Why is "fetch from" removed here?

Reply:
Passing by and saw this. Thought of clearing this misconception - You don't need mockery or other such tools if you're in ES6 land. Its easy to mock any module within the test file using plain old JS. You need to import the module being mocked first, then the file under test.